dev/io.openliberty.grpc.1.0.internal.client/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -18,29 +18,42 @@
 clientConfig=GRPC Client Properties
 clientConfig.desc=Configuration properties to be applied to gRPC targets that match the specified URI.
 
-target=Remote target
-target.desc=The remote gRPC target URI, with wildcard support.
+host=Remote host
+host.desc=The remote gRPC service hostname or IP address, with wildcard support.
+
+path=Remote procedure path
+path.desc=The remote gRPC service path, with wildcard support. A path consists of the gRPC service and method names, in the form "ServiceName/MethodName".
 
 headersToPropagate=HTTP header names to propagate
 headersToPropagate.desc=HTTP header names to propagate from an inbound request to an outbound call.
 
 clientInterceptors=GRPC client interceptors
 clientInterceptors.desc=A list of fully qualified class names for gRPC client interceptor classes.
 
-authnToken=Authorization token
-authnToken.desc=The type of authorization token to propagate. Currently supported token types are jwt or oauth.
-
-enableKeepAlive=Keepalive
-enableKeepAlive.desc=Enable gRPC keepalive.
-
 keepAliveTime=Keepalive interval
 keepAliveTime.desc=The time to wait for new messages before sending a new keepalive ping.
 
 keepAliveTimeout=Keepalive timeout
 keepAliveTimeout.desc=End the connection if a keepalive response is not received within this time.
 
+keepAliveWithoutCalls=Keepalive without calls
+keepAliveWithoutCalls.desc=Perform keepalive when there are no outstanding RPCs on the channel.
+
 maxInboundMessageSize=Maximum message size
 maxInboundMessageSize.desc=The maximum inbound message size.
  
-sslConfig=The id of an ssl reference in server.xml
-sslConfig.desc=An id of an ssl reference found in server.xml. The ssl configuration that is specified by this reference is used in the gRPC client configuration.
+sslRef=SSL Reference
```

Comment:
Suggested update

>SSL Reference

To

>SSL reference

Reply:
I agree that making reference lowercase here makes the most sense - but the build process includes a schema validator tool that fails if I make this change.

Reply:
`junit.framework.AssertionFailedError: The label "SSL reference" for element ssl of pid io.openliberty.grpc.clientConfig should be in title case.`

Reply:
Ah, okay. Then it's alright.